### ISSUE TITLE
C# does not work on windows

### ISSUE THREAD

**Issue by PascalSenn** [onivim/oni2#2375]
I could not get C# working on windows. Is there a way to see the logs of the extensions?
also, will there be hover support or is this a limitation?

**Comment by bryphe** [comment]
Hi @PascalSenn - hmm, what sort of C# project are you using (Console App, Unity, etc) - is it open source? And just to confirm - which C# extension are you using?

The omnisharp server tools tend to work best with the dotnet core SDK - do you have the `dotnet` command available in your `PATH`?

> Is there a way to see the logs of the extensions?

This would help, for sure. The extensions log some output - we don't have a UI for it, yet - tracking here: https://github.com/onivim/oni2/issues/2365 In the meantime, some of it does come through when running `esy run -f -debug` or to a log file - `esy run -f -debug --log-file=onivim2.log`.

I'd expect the first time you open the project after installing the extension, that you'd see this UI in the statusbar:

![image](https://user-images.githubusercontent.com/13532591/91647769-491ae900-ea13-11ea-90b9-8ab97abe1b54.png)

And then, once it's set up, the C# extension shows the current project and server status in the status bar:
![image](https://user-images.githubusercontent.com/13532591/91647773-5a63f580-ea13-11ea-9527-d23385044b7c.png)

This is the recommended extension to use, as well:
![image](https://user-images.githubusercontent.com/13532591/91647783-8bdcc100-ea13-11ea-85bc-839c52d6bb9b.png)

Let me know what sort of project you were using, and I'll see if I can reproduce on my Windows machine.

**Comment by PascalSenn** [comment]
Thanks for the detailed response!
Tried it on my linux machines and it's working perfectly! 

I think i corrupted my windows enthronement by installing extensions that are not officially supported. I removed them but i still see issues. 

I will close this issue an reopen in case the problem persists. 

Thank you for your help. Looking forward to replace vscode! 
Only a couple of features are missing to do so!
